libcodechecker/profiler.py

```diff
@@ -19,6 +19,25 @@
 LOG = LoggerFactory.get_new_logger('PROFILER')
 
 
+class Timer(object):
+    """
+    Simple timer context manager
+    to measure code block execution time.
+    """
+    def __init__(self, block_name=''):
+        self.before = None
+        self.after = None
+        self.block_name = block_name
+
+    def __enter__(self):
+        self.before = datetime.datetime.now()
+
+    def __exit__(self, type, value, traceback):
+        after = datetime.datetime.now()
```

Comment:
I think either this should be `self.after` or `self.after` should not be initialized.

Reply:
I've cleaned it up.